components/masking_functions/include/masking_functions/sql_context.hpp

```diff
@@ -30,6 +32,8 @@ namespace masking_functions {
 // construction.
 class sql_context {
  public:
+  using dict_container_type = std::multimap<std::string, std::string>;
```

Comment:
In my naming convention `_container` is already a type suffix (similarly to `_type`) meaning this is a container(vector, list, map, etc.) of something. In other words, if somebody sees a type whose name ends with `_container`, he can make certain assumptions.
So, here I would name it `dictionary_term_container`.

Reply:
Done